src/modules/playerctl.rs

```diff
@@ -0,0 +1,163 @@
+        .output()
+        .await;
+
+    match get_current_song_cmd {
+        Ok(get_current_song_cmd) => String::from_utf8_lossy(&get_current_song_cmd.stdout)
+            .trim()
+            .into(),
+        Err(e) => {
+            error!("Error: {:?}", e);
+            String::new()
+        }
+    }
+}
+
+fn get_volume() -> f64 {
+    let get_current_song_cmd = std::process::Command::new("bash")
+        .arg("-c")
+        .arg("playerctl volume")
+        .stdout(Stdio::piped())
+        .output();
+
+    match get_current_song_cmd {
+        Ok(check_update_cmd) => {
+            String::from_utf8_lossy(&check_update_cmd.stdout)
+                .trim()
+                .parse::<f64>()
+                .unwrap()
+                * 100.0
+        }
+        Err(e) => {
+            error!("Error: {:?}", e);
+            100.0
+        }
+    }
+}
+
+pub struct Playerctl {
+    song: String,
+    volume: f64,
+}
+
+impl Default for Playerctl {
+    fn default() -> Self {
+        Self {
+            song: String::default(),
+            volume: get_volume(),
+        }
+    }
+}
+
+#[derive(Debug, Clone)]
+pub enum Message {
+    SetSong(String),
+    Prev,
+    Play,
+    Next,
+    SetVolume(f64),
+}
+
+impl Playerctl {
+    pub fn update(&mut self, message: Message) -> Task<crate::app::Message> {
+        match message {
+            Message::SetSong(song) => {
+                self.song = song;
+                Task::none()
+            }
+            Message::Prev => {
+                execute_command("playerctl previous".to_string());
+                Task::perform(async move { get_current_song().await }, move |song| {
+                    app::Message::Playerctl(Message::SetSong(song))
+                })
+            }
+            Message::Play => {
+                execute_command("playerctl play-pause".to_string());
+                Task::perform(async move { get_current_song().await }, move |song| {
+                    app::Message::Playerctl(Message::SetSong(song))
+                })
+            }
+            Message::Next => {
+                execute_command("playerctl next".to_string());
+                Task::perform(async move { get_current_song().await }, move |song| {
+                    app::Message::Playerctl(Message::SetSong(song))
+                })
+            }
+            Message::SetVolume(v) => {
+                execute_command(format!("playerctl volume {}", v / 100.0));
+                self.volume = v;
+                Task::none()
+            }
+        }
+    }
+
+    pub fn menu_view(&self) -> Element<Message> {
+        column![
+            slider(0.0..=100.0, self.volume, |new_v| {
+                Message::SetVolume(new_v)
+            }),
+            row![
+                button(icon(Icons::SkipPrevious)).on_press(Message::Prev),
+                button(icon(Icons::PlayPause)).on_press(Message::Play),
+                button(icon(Icons::SkipNext)).on_press(Message::Next)
```

Comment:
```suggestion
                button(icon(Icons::SkipPrevious))
                    .on_press(Message::Prev)
                    .padding([5, 12])
                    .style(SettingsButtonStyle.into_style()),
                button(icon(Icons::PlayPause))
                    .on_press(Message::Play)
                    .style(SettingsButtonStyle.into_style()),
                button(icon(Icons::SkipNext))
                    .on_press(Message::Next)
                    .padding([5, 12])
                    .style(SettingsButtonStyle.into_style())
```

May I suggest using this style to align with the other button in the settings menu?